fretwell-distributed-government.md

```diff
@@ -1,98 +1,93 @@
+In this chapter ... 
 
-## Why distributed is important
+## Defining distributed
+In his excellent book on the subject, [Distributed Teams: The Art and Practice of Working Together While Physically Apart](https://www.amazon.com/dp/1732254907/ref=cm_sw_su_dp), author John O'Duinn, explains why distributed teams are important and how they can be even more effective than their physically collocated counterparts.
 
-Distributed takes a holistic approach to work and what society values, and areas civic institutions must address if they are to stay relevant.
+While people may flippantly use antiquated terms like "telework" and "remote," O'Duinn makes clear [distinctions](https://oduinn.com/2020/02/25/distributed-team-vs-remote-work-and-work-from-home/) between key terms, such as "distributed teams," "virtual teams," "virtual employee," "remote work," "remote employee," "work from home," "work from anywhere" and "telework."
 
-Here are key reasons why distributed teams are critical, especially for government:
+And the subtle differences in each drastically impact our perceptions of and assumptions about distributed teams.
 
-* Representation: Geographically distributed government workforce more closely matches the needs and concerns of the wider population.
+O'Duinn describes "distributed" this way:
 
-* Resiliency: Ability to maintain operations regardless of whereabouts, especially if physical locations or regions have been compromised for a prolonged period of time.
+> All humans on the team work together, even though they are physically apart from each other. This is not a collection of individuals who each do solo heads-down work from different locations. Instead, this is a group of humans who coordinate their work with others on their physically distributed team. Because everyone on the physically distributed team is "remote" from someone, it is clear that everyone on the team has equal responsibility to communicate and coordinate their work with coworkers—regardless of whether any individual human is working from a building with the company logo on the door, from home, from a coworking space, a hotel or a parked car! Example usage: "I work on a distributed team," "my team is distributed."
 
-* Recruiting: Talent pool is expanded to include people of diverse backgrounds, ages, abilities and experience.
+_XXX_
```

Comment:
Placeholder for potential material: Why is this distinction important to both you, the author, and to your intended readers? This would be a nice place to answer that question.